deploy/olm-catalog/keycloak-operator/7.0.1/keycloak-operator.v7.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,242 @@
+      mediatype: image/svg+xml
+  install:
+    spec:
+      deployments:
+      - name: keycloak-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: keycloak-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: keycloak-operator
+            spec:
+              containers:
+              - command:
+                - keycloak-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: keycloak-operator
+                image: quay.io/keycloak/keycloak-operator:latest
```

Comment:
This should be locked down to the operator release 7.0.1, right?

Reply:
Yes, we don't have that tag yet but I will update the CSV.